sql/testdata/builtin_function

```diff
@@ -254,16 +254,70 @@ SELECT now() - current_date::timestamp < interval '24h10s'
 ----
 true
 
-query error syntax error at or near "\("
-SELECT current_date()
+query T
+SELECT current_date() - transaction_date() = interval '0s'
+----
+true
+
+query T
+SELECT current_date - transaction_date = interval '0s'
+----
+true
+
+query T
+SELECT now() - current_timestamp = interval '0s'
+----
+true
+
+query T
+SELECT now() - statement_timestamp = interval '0s'
+----
+true
+
+query T
+SELECT current_timestamp() - statement_timestamp() = interval '0s'
```

Comment:
It would be nice if we could have some tests where statement_timestamp and transaction_timestamp differ, but I don't see how to do that without introducing ugly test-only hooks.